platform/darwin/src/MGLStyle.h

```diff
@@ -484,24 +521,40 @@ MGL_EXPORT
 #pragma mark Managing Style Classes
 
 /**
- Support for style classes has been removed. This property always returns an empty array.
+ Currently active style classes, represented as an array of string identifiers.
  */
-@property (nonatomic) NS_ARRAY_OF(NSString *) *styleClasses __attribute__((deprecated("This property is non-functional.")));
```

Comment:
I will keep the version from master so this PR will not show any changes on that file.